hcache/hcache.c

```diff
@@ -343,7 +343,7 @@ static void restore_address(struct Address **a, const unsigned char *d, int *off
     restore_char(&(*a)->personal, d, off, convert);
     restore_char(&(*a)->mailbox, d, off, false);
     restore_int(&g, d, off);
-    (*a)->group = g;
+    (*a)->group = g ? true : false;
```

Comment:
Why not just add a comment here instead of introducing a noop and relying on the compiler to optimize it away?
even `(*a)->group = g /* ? true : false */;` would be possible.

Reply:
btw: as `group` is boolean, can we rename it  to e.g. `hasGroup` ?

Reply:
@toogley 
> `hasGroup`

no, no, NO!  That's terribly un-c!  `has_group` is much better :-)

I'll raise an issue (tomorrow) to discuss our options.

Reply:
Sorry, I cannot resist: How about `bHasGroup`?

Reply:
Not sure if you guys *tried* to point out my comment as bikeshedding or if that's just the natural progression when discussing these kinds of things... :laughing: